### PR TITLE
Support helm v3

### LINKED ISSUE #479: Failed to install mysql-operator with helm3
```
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
NAME: mysql-operator
LAST DEPLOYED: Wed Feb 19 17:00:54 2020
NAMESPACE: mysql-operator
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
You can create a new cluster by issuing:

cat <<EOF | kubectl apply -f-
apiVersion: mysql.presslabs.org/v1alpha1
kind: MysqlCluster
metadata:
  name: my-cluster
spec:
  replicas: 1
  secretName: my-cluster-secret
---
apiVersion: v1
kind: Secret
metadata:
  name: my-cluster-secret
type: Opaque
data:
  ROOT_PASSWORD: $(echo -n "not-so-secure" | base64)
EOF
```
 crds were not installed
```
bgpconfigurations.crd.projectcalico.org       2020-02-17T07:08:47Z
bgppeers.crd.projectcalico.org                2020-02-17T07:08:47Z
blockaffinities.crd.projectcalico.org         2020-02-17T07:08:47Z
chartrepos.app.alauda.io                      2020-02-19T06:59:48Z
charts.app.alauda.io                          2020-02-19T06:59:48Z
clusterinformations.crd.projectcalico.org     2020-02-17T07:08:48Z
felixconfigurations.crd.projectcalico.org     2020-02-17T07:08:47Z
globalnetworkpolicies.crd.projectcalico.org   2020-02-17T07:08:48Z
globalnetworksets.crd.projectcalico.org       2020-02-17T07:08:48Z
helmcharts.helm.kubedex.com                   2020-02-19T05:22:49Z
helmrequests.app.alauda.io                    2020-02-19T06:59:48Z
hostendpoints.crd.projectcalico.org           2020-02-17T07:08:48Z
ipamblocks.crd.projectcalico.org              2020-02-17T07:08:47Z
ipamconfigs.crd.projectcalico.org             2020-02-17T07:08:47Z
ipamhandles.crd.projectcalico.org             2020-02-17T07:08:47Z
ippools.crd.projectcalico.org                 2020-02-17T07:08:48Z
networkpolicies.crd.projectcalico.org         2020-02-17T07:08:49Z
networksets.crd.projectcalico.org             2020-02-17T07:08:49Z
releases.app.alauda.io                        2020-02-17T08:35:21Z

```

### PR DESCRIPTION
**Pull request by AMecea**
This PR contains commits from #441 and #445.

Also this PR bumps `controller-tools` dependency.

Fixes #479 and #433

### CODE REVIEW

File: hack/generate_chart_manifests.sh [hunk 1]
```diff
@@ -3,22 +3,16 @@
...
 
-# templates/crds.yaml update
-awk 'FNR==1 && NR!=1 {print "---"}{print}' ${CONFIG_PATH}/crds/*.yaml > ${CHART_PATH}/templates/_crds.yaml
-yq m -d'*' -i ${CHART_PATH}/templates/_crds.yaml chart-metadata.yaml
-yq w -d'*' -i ${CHART_PATH}/templates/_crds.yaml 'metadata.annotations[helm.sh/hook]' crd-install
-yq d -d'*' -i ${CHART_PATH}/templates/_crds.yaml metadata.creationTimestamp
-yq d -d'*' -i ${CHART_PATH}/templates/_crds.yaml status
-yq d -d'*' -i ${CHART_PATH}/templates/_crds.yaml spec.validation
+# crds/crds.yaml update
+awk 'FNR==1 && NR!=1 {print "---"}{print}' ${CONFIG_PATH}/crds/*.yaml > ${CHART_PATH}/crds/crds.yaml
```
Comment (calind) [inline hack/generate_chart_manifests.sh]: Now we can copy each CRD file individually.

File: charts/mysql-operator/README.md [hunk 2]
```diff
@@ -4,8 +4,13 @@ This is the helm chart for [mysql-operator](https://github.com/presslabs/mysql-o
 
 ## TL;DR
 ```sh
+## For Helm v2
 helm repo add presslabs https://presslabs.github.io/charts
 helm install presslabs/mysql-operator --name mysql-operator
+
+## For Helm v3
+helm repo add presslabs https://presslabs.github.io/charts
+helm install mysql-operator presslabs/mysql-operator --skip-crds
```
Comment (calind) [inline charts/mysql-operator/README.md]:
```suggestion
helm install mysql-operator presslabs/mysql-operator
```

Just mention in the docs.
